src/theme/overrides/MuiButton.js

```diff
@@ -1,49 +1,62 @@
 import palette from '../palette'
 
-const MuiButton = {
-  contained: {
-    boxShadow: 'none',
-    '&:disabled': {
-      backgroundColor: palette.primary.main,
-      color: palette.common.white,
-      opacity: 0.5,
-    },
-    '&:hover': {
+const MuiButton = (color) => {
+  const customColor = (color || 'primary')
```

Comment:
I think this part could be omitted as now the color checking happens inside Button